packages/frontend/editor-ui/src/components/canvas/experimental/components/ExperimentalEmbeddedNdvMapper.vue

```diff
@@ -23,27 +25,52 @@ const vf = useVueFlow();
 const canvas = inject(CanvasKey, undefined);
 const isVisible = computed(() => visible && !canvas?.isPaneMoving.value);
 const isOnceVisible = ref(isVisible.value);
+const canvasStore = useCanvasStore();
+const contentElRef = computed(() => contentRef.value?.$el ?? null);
+const contentSize = useElementSize(contentElRef);
+const refBounding = useElementBounding(virtualRef);
 
-watchOnce(isVisible, (value) => {
-	isOnceVisible.value = isOnceVisible.value || value;
+watch(
+	isVisible,
+	(value) => {
+		isOnceVisible.value = isOnceVisible.value || value;
+		canvasStore.setSuppressInteraction(value);
+	},
+	{ immediate: true },
+);
+
+onBeforeUnmount(() => {
+	canvasStore.setSuppressInteraction(false);
 });
 
 defineExpose({
-	contentRef: computed<HTMLElement>(() => contentRef.value?.$el ?? null),
+	contentRef: contentElRef,
 });
 </script>
 
 <template>
-	<N8nPopover
+	<ElPopover
```

Comment:
Switching to ElPopover because N8nPopover's wrapper `<span />` element causes layout bug when used in a flexbox container.